--- SLmake.inc	2002-05-28 17:09:42.000000000 +0200
+++ SLmake.inc	2002-05-28 17:09:08.000000000 +0200
@@ -19,7 +19,7 @@
 #  The complete path to the top level of ScaLAPACK directory, usually
 #  $(HOME)/SCALAPACK
 #
-home          = $(HOME)/SCALAPACK
+home          = @SCALAPACK_HOME@
 #
 #  The platform identifier to suffix to the end of library names
 #
@@ -29,16 +29,16 @@
 #  and the directory where the BLACS libraries are
 #
 BLACSDBGLVL   = 0
-BLACSdir      = /usr/local/lib
+BLACSdir      = /usr/lib
 #
 #  MPI setup; tailor to your system if using MPIBLACS
 #  Will need to comment out these 6 lines if using PVM
 #
 USEMPI        = -DUsingMpiBlacs
-SMPLIB        = /usr/local/mpich-1.2.1/lib/libmpich.a
-BLACSFINIT    = $(BLACSdir)/libmpiblacsF77init-p4.a
-BLACSCINIT    = $(BLACSdir)/libmpiblacsCinit-p4.a
-BLACSLIB      = $(BLACSdir)/libmpiblacs-p4.a
+SMPLIB        = /usr/lib/libmpich.a
+BLACSFINIT    = $(BLACSdir)/blacsF77init_MPI-LINUX-0.a
+BLACSCINIT    = $(BLACSdir)/blacsCinit_MPI-LINUX-0.a
+BLACSLIB      = $(BLACSdir)/blacs_MPI-LINUX-0.a
 TESTINGdir    = $(home)/TESTING
 
 #
